src/Avalonia.SceneGraph/Media/Imaging/IBitmap.cs

```diff
@@ -30,5 +31,11 @@ public interface IBitmap
         /// </summary>
         /// <param name="fileName">The filename.</param>
         void Save(string fileName);
+
+        /// <summary>
+        /// Saves the bitmap to a stream in png format.
+        /// </summary>
+        /// <param name="stream">The stream.</param>
+        void Save(Stream stream);
```

Comment:
This should probably accept an image format somehow, but it's fine just writing to .png at the moment. Maybe document that fact in the `IBitmapImpl` doc comments too?